src/ui/common/src/components/pages/artifact/id/index.tsx

```diff
@@ -143,6 +154,27 @@ const ArtifactDetailsPage: React.FC<ArtifactDetailsPageProps> = ({
               contentWithLoadingStatus={contentWithLoadingStatus}
             />
           </Box>
+          <Box display="flex" width="100%" paddingTop="40px">
+            <Box width="100%">
+              <OperatorSummaryList
```

Comment:
How does this differ from `ArtifactSummaryList` and can we reuse/create a more generalizable component for both purposes?

Reply:
We should create a component which just combines the two and either returns an OperatorSummaryList or ArtifactSummaryList based on a prop that you pass in.

This is a nice to have. We can address this work in a follow up task.

Reply:
See linear task here: https://linear.app/aqueducthq/issue/ENG-1768/create-reusable-component-for-artifactoperator-summary-list